app/scripts/lib/mailcheck.js

```diff
@@ -9,6 +9,12 @@ define(function (require, exports, module) {
 
   const _ = require('underscore');
   const $ = require('jquery');
+
+  if (! window.jQuery) {
+    // This is needed to ensure the mailcheck plugin can attach itself.
+    window.jQuery = $;
+  }
```

Comment:
How come we need to do this? Because further up the changeset I see this:

https://github.com/mozilla/fxa-content-server/pull/5868/files#diff-9907ccc3040dddc91436c0abe3224854R9

Belt and braces?